ted_sws/core/adapters/logger.py

```diff
@@ -0,0 +1,82 @@
+import abc
+import enum
+import logging
+
+import logstash
+
+from ted_sws import config
+
+
+class LoggingType(enum.Enum):
+    ELK = "ELK"
+    PY = "PY"
+
+
+DOMAIN_LOGGING_TYPES = config.LOGGING_TYPE.split(",") if config.LOGGING_TYPE is not None else [LoggingType.PY.value]
+DOMAIN_LOGGING_TYPE = DOMAIN_LOGGING_TYPES[0]
+
+
+class LoggerABC(abc.ABC):
+    """
+    This abstract class provides methods definitions and infos for available loggers
+    """
+    pass
+
+
+class Logger(LoggerABC):
+    """
+    This class provides common features for available loggers
+    """
+
+    def __init__(self, name: str = __name__):
+        self.name = name
+        self.logger = logging.getLogger(name)
+        self.level = logging.DEBUG
+        self.logger.setLevel(self.level)
+
+    def get_logger(self) -> logging.Logger:
+        return self.logger
+
+    def log(self, msg: str):
```

Comment:
Shouldn't we be logging message objects? 
Are they Strings now? 

I was dreaming of separating the Technical from Domain events. But probably this was overlooked even in my specifications.

Reply:
This method is for logging string message to final destination (console, file, etc). The object/event transformation into loggable string is handled by corresponding message_handler